templates/artist/artist_application_thanks.html

```diff
@@ -3,6 +3,7 @@
 {% block content %}
     <h2>Artist Application</h2>
     <div class="page-padding">
-        <p>Thank you! Your application was received successfully.</p>
+        <h4>Thank you! Your submission was received successfully.</h4>
+        <p class="extra-padding">We will be contacting you soon with further details on launching your campaign.If you haven't already, please look over the <strong><a href="{% url 'resources' %}" target=" _blank">Artist Resources</a></strong> page and reach out to us <strong><a href="{% url 'contact' %}" target=" _blank">here</a></strong> if you have any additional questions.</p>
```

Comment:
Couple nits on this line:
- As before, remove space before `_blank` for link target
- As before, Use `<strong>` tags inside of `<a>` tags
- Add space between sentences
- Consider tying the link for the contact page to the verb to contact, rather than having a link on the `here` text (reason for this is that emphasis is being put on the text `here`, but that is not helpful when scanning through the page)

Regarding that last point, imagine we replace the text that is linked with all caps:
> ...and reach out to us HERE if you have any additional questions.

vs.

> ...and REACH OUT TO US if you have any additional questions.

If you think about the link in this way, it can help to make links more useful when a user is skimming through at a glance.

```suggestion
        <p class="extra-padding">We will be contacting you soon with further details on launching your campaign. If you haven't already, please look over the <a href="{% url 'resources' %}" target="_blank"><strong>Artist Resources</strong></a> page and <a href="{% url 'contact' %}" target="_blank"><strong>reach out to us</strong></a> if you have any additional questions.</p>
```

Reply:
Totally agree. That looks a lot better